docs/PACKAGE_METADATA_FOR_EXECUTABLE_FILES.md

```diff
@@ -82,9 +83,11 @@ types and ranges.
 
 Reference implementations of [packaging tools for .deb and .rpm](https://github.com/systemd/package-notes)
 are available, and provide macros/helpers to include the note in binaries built
-by the package build system. They make use of the new `--package-metadata` flag that
-is available in the bfd, gold, mold and lld linkers (versions 2.39, 1.3.0 and 15.0
-respectively). This linker flag takes a JSON payload as parameter.
+by the package build system.
+They make use of the new `--package-metadata` flag that is available in the
```

Comment:
the flag should be suffixed with `=` btw, according to our usual spelling for switches that take parameters.